src/test/scala/com/github/pjfanning/pekkobuild/PekkoCoreDependency.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
```

Comment:
none of this code seems to originate from Lightbend

Reply:
Let me change it, sec

Reply:
Fixed and pushed.